added logstash info (#53)
opensearch-project/documentation-website: _clients/logstash/index.md

* `filter` parses and enriches the events in one way or the other. Logstash has a large collection of filter plugins that modify events and pass them on to an output. For example, a `grok` filter parses unstructured events into fields and a `mutate` filter changes fields. Filters are executed sequentially.
* `output` ships the filtered events to one or more destinations. Logstash supports a wide range of output plugins for destinations like OpenSearch, TCP/UDP, emails, files, stdout, HTTP, Nagios, and so on.

Both the input and output phases support codecs to process events as they enter or exit the pipeline.
Some of the popular codecs are `json` and `multiline`. The `json` codec processes data that’s in JSON format and the `multiline` codec merges multiple line events into a single line.

You can also write conditional statements within pipeline configurations to perform certain actions, if a certain criteria is met.

> ## Install Logstash on MAC / Linux

Reviewer: For GA we will be supporting only Linux

Author: Fixed.

Make sure you have [Java Development Kit (JDK)](https://www.oracle.com/java/technologies/javase-downloads.html) version 8 or 11 installed.

1. Download the Logstash tarball from [OpenSearch downloads](https://opensearch.org/downloads.html).

2. Navigate to the downloaded folder in the terminal and extract the files:

     ```bash

>      tar -zxvf logstash-7.13.2-darwin-x86_64.tar.gz

Reviewer: logstash-oss-with-opensearch-output-plugin-7.13.2-macos-x64.tar.gz

## Install Logstash

The OpenSearch Logstash plugin has two installation options at this time: Linux (ARM64/X64) and Docker (ARM64/X64).

Make sure you have [Java Development Kit (JDK)](https://www.oracle.com/java/technologies/javase-downloads.html) version 8 or 11 installed.

### Tarball

1. Download the Logstash tarball from [OpenSearch downloads](https://opensearch.org/downloads.html).

2. Navigate to the downloaded folder in the terminal and extract the files:

     ```bash
     tar -zxvf logstash-oss-with-opensearch-output-plugin-7.13.2-linux-x64.tar.gz
     ```

3. Navigate to the `logstash-7.13.2` directory.
- You can add your pipeline configurations to the `config` directory. Logstash saves any data from the plugins in the `data` directory. The `bin` directory contains the binaries for starting Logstash and managing plugins.

### Docker

1. Pull the Logstash oss package with the OpenSearch output plugin image:

```
docker pull opensearchproject/logstash-oss-with-opensearch-output-plugin:7.13.2
```

1. Create a Docker network:

```
docker network create test
```

1. Start OpenSearch with this network:

```

> docker run -p 9200:9200 -p 9600:9600 --name opensearch --net mynet -e "discovery.type=single-node" opensearchproject/opensearch:1.0.0

Reviewer: ```suggestion
docker run -p 9200:9200 -p 9600:9600 --name opensearch --net test -e "discovery.type=single-node" opensearchproject/opensearch:1.0.0
```

```

1. Start Logstash:

```

> docker run -it --rm --name logstash1 --net mynet opensearchstaging/logstash-oss-with-opensearch-output-plugin:7.13.2 -e 'input { stdin { } } output {

Reviewer: ```suggestion
docker run -it --rm --name logstash --net test opensearchstaging/logstash-oss-with-opensearch-output-plugin:7.13.2 -e 'input { stdin { } } output {
```

  opensearch {
    hosts => ["https://opensearch:9200"]

>     index => "opensearch-logstash-mac-x64-docker-%{+YYYY.MM.dd}"

Reviewer: ```suggestion
    index => "opensearch-logstash-docker-%{+YYYY.MM.dd}"
```

1. Start OpenSearch with this network:

```
docker run -p 9200:9200 -p 9600:9600 --name opensearch --net test -e "discovery.type=single-node" opensearchproject/opensearch:1.0.0
```

1. Start Logstash:

```

> docker run -it --rm --name logstash --net test opensearchstaging/logstash-oss-with-opensearch-output-plugin:7.13.2 -e 'input { stdin { } } output {

Reviewer: ```suggestion
docker run -it --rm --name logstash --net test openserachproject/logstash-oss-with-opensearch-output-plugin:7.13.2 -e 'input { stdin { } } output {
```
